packages/app/src/app/pages/Dashboard/Content/CreateNewSandbox/Modal/Components/MyTemplatesTab.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { useQuery } from '@apollo/react-hooks';
+import styled from 'styled-components';
+import Template from '@codesandbox/common/lib/components/Template';
+import { LIST_TEMPLATES } from '../../../../queries';
+
+import { Templates } from '../elements';
+
+const Title = styled.h3`
+  font-family: 'Poppins';
+  font-weight: 600;
+  width: 288px;
+  font-size: 24px;
+  line-height: 36px;
+
+  color: ${props => props.theme.white};
+`;
+
+const Empty = styled.div`
+  width: 100%;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`;
+
```

Comment:
Let's move all styled components into `elements.js` before merging this. I can make a pass over the code to apply conventions if that sounds good to you.

Reply:
Moved :)